### PR TITLE
Feat carrousel swiper

### PR DESCRIPTION
**Pull request by BeAmarawn**
### Summary

- Added a name to android package
- Update on some dependencies related with React Native Reanimated
- Created an interface to represent input user data
- Created a custom Card Component
- Created a native Deck Swiper animation, feeded by any array following the profile's interface, each Card when is swiped has a -- Callback function to remove it from the list and can perform another action.

### Changelog
- Added a android package name on 'packages/expo/app.json'
- Added a mock images on 'packages/ui/assets/images/examples'
- Added an Profile interface on 'packages/ui/@types'
- Added a custom card component on 'packages/ui/core/Card'
- Added a native custom  Deck Swiper animation on 'packages\expo\src\modules\home\Home.tsx'

### Specific modules used
[React Native Reanimated](https://github.com/software-mansion/react-native-reanimated)
[React Native Gesture Handler](https://github.com/software-mansion/react-native-gesture-handler)

Screenshot3
Screenshot2
Screenshot1

https://user-images.githubusercontent.com/38305896/104603715-716e5180-565b-11eb-922f-49dd40aaf8f6.mp4

### CODE REVIEW

**Comment by belgamo** [comment]
Please rebase your branch.

File: packages/expo/src/modules/home/Home.tsx [hunk 1]
```diff
@@ -1,23 +1,246 @@
...
+    image: require('../../../../ui/assets/images/examples/dog4.jpg'),
+  },
+];
+
+//Mathematical equations to calculate the trigonometric coefficient of rotation
+const {width, height} = Dimensions.get('window');
+const rotatedWidth = width * Math.sin((75 * Math.PI) / 180) + height * Math.sin((15 * Math.PI) / 180);
+
+//Animation function to spring animation
+function runSpring(
```
Comment (brunohkbx) [inline packages/expo/src/modules/home/Home.tsx]: What about moving all the animation setup to another file?

Comment (BeAmarawn) [reply packages/expo/src/modules/home/Home.tsx]: I liked it, where would you suggest put the file with the config?

Comment (brunohkbx) [reply packages/expo/src/modules/home/Home.tsx]: I think under this directory should be fine.

**Comment by belgamo** [comment]
Closing this because it's been a while since this PR was opened and rebase process seems a little messy. @BeAmarawn I'm gonna put this issue back on the to-do list again. Please let me know if you are interested in contributing to this issue remaking this PR.

Thanks a lot!

**Pull request closed without merging**